UI.PageModels/Pages/VhVideoWebPage.cs

```diff
@@ -22,4 +21,14 @@ public VideoWebLogoutPage SignOut(bool confirmSignOut = true)
 
         return new VideoWebLogoutPage(Driver, DefaultWaitTime);
     }
+
+    public void OpenChatWithVHO() => ClickElement(By.XPath("//img[@id='open-chat-arrow']"));
```

Comment:
This probably belongs to the waiting room page. If there isn't a `WaitingRoomPage` this might be a good time to add it